packages/orbit-components/src/BadgeList/BadgeListItem/index.js

```diff
@@ -0,0 +1,110 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import Text, { StyledText } from "../../Text";
+import TYPE_OPTIONS from "../consts";
+import defaultTheme from "../../defaultTheme";
+import { ICON_COLORS } from "../../Icon/consts";
+import { StyledTooltipChildren } from "../../primitives/TooltipPrimitive";
+import { right } from "../../utils/rtl";
+
+import type { Props } from "./index";
+
+const getBackground = ({ theme, $type }) => {
+  const tokens = {
+    [TYPE_OPTIONS.NEUTRAL]: theme.orbit.paletteCloudLight,
+    [TYPE_OPTIONS.INFO]: theme.orbit.paletteBlueLight,
+    [TYPE_OPTIONS.SUCCESS]: theme.orbit.paletteGreenLight,
+    [TYPE_OPTIONS.WARNING]: theme.orbit.paletteOrangeLight,
+    [TYPE_OPTIONS.CRITICAL]: theme.orbit.paletteRedLight,
+  };
+  return tokens[$type];
+};
+
+const getIconColor = type => {
+  if (type === TYPE_OPTIONS.NEUTRAL) return ICON_COLORS.SECONDARY;
+  return type;
+};
+
+const StyledBadgeListItem = styled.li`
```

Comment:
hm, probably you do not need here custom styled component, it can be replaced with: 
```
    <Stack as="li" spaceAfter="smallest" align="center" spacing="none" data-test={dataTest}>
       ... content
    </Stack>
```

Reply:
If we gonna use Stack for this use case, we need to also reset/normalize the styles for the `li` element, because it will have the default user agent styles. That's the reason why I've chosen this approach. Don't want to add yet another complexity to Stack 🙂